feat(snippet-bot): allow `UNTRACKED` SnippetStatus

tmatsuo opened the pull request:
fixes #1244

tmatsuo commented:
> Here is the summary of possible violations 😱

@snippet-bot It's ok friend :)

chingor13 reviewed (commented):
Can we add a test for this so we can show what the bot is or isn't doing for these?

tmatsuo commented:
@chingor13 I updated the fixture and snapshot to do that.
We already have cases with and without warnings. I added an additional warning to the existing snapshot.

Technically we can have another test case, but I think the current test suffice. Let me know if you think differently.
